src/components/cylc/table/Table.vue

```diff
@@ -240,40 +227,59 @@ export default {
         mdiChevronDown,
         mdiArrowDown
       },
-      sortBy: ['Cycle Point'],
+      sortBy: ['node.cyclePoint'],
       sortDesc: [localStorage.cyclePointsOrderDesc ? JSON.parse(localStorage.cyclePointsOrderDesc) : true],
       expanded: [],
       headers: [
         {
-          text: 'Task'
+          text: 'Task',
+          value: 'node.name',
+          sort: DEFAULT_COMPARATOR
         },
         {
           text: 'Jobs',
-          value: 'data-table-expand'
+          value: 'data-table-expand',
+          sortable: false
         },
         {
-          text: 'Cycle Point'
+          text: 'Cycle Point',
+          value: 'node.cyclePoint',
+          sort: (a, b) => DEFAULT_COMPARATOR(String(a ?? ''), String(b ?? ''))
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
This looks good